src/components/Forms/LoginForm.js

```diff
@@ -60,25 +59,27 @@ const LoginForm = ({ handleChange, data, onSubmit, loading }) => (
       <Button
         block
         disabled={loading}
-        onPress={onSubmit}
+        onPress={() => onSubmit(false)}
```

Comment:
in order to avoid using lambdas in JSX how about this instead?

in LoginScreen.js:

`onSubmit = (isFb = false) => {`

then no change in this file.

Reply:
@JoshFabian good suggestion.